src/_content/posts/djangocon-us-grants.md

```diff
@@ -0,0 +1,63 @@
+
+
+Important Dates to Remember:
+- Application Deadline: **May 4, 2025**, at Noon Eastern Daylight Time (America/New_York).
+- Decision Notifications: Will be sent out by **June 23, 2025**.
+- Recipients of Opportunity Grants will receive separate instructions on how to upload necessary receipts and provide their payment information to facilitate reimbursement.
+- Reimbursements are made a week after the conference, which typically lasts 4 to 5 weeks, to consolidate. Be patient, as the team is working tirelessly to get everyone sorted. In case of questions during that period, reach out to us via [email](mailto:grants@djangocon.us)
+
+The dates are subject to change based on the review process and the number of applications.
+
+
+Attending a conference can be costly, and our [Opportunity Grants](https://docs.google.com/forms/d/e/1FAIpQLSfuriCI4JtAEcTfLy9Qdk2h_DfmaGKE544svRfMJLCOu5eLDQ/viewform) aim to relieve some of that financial burden. We strive to support attendees based on their individual needs. While we may not be able to fulfil every request in full, we are committed to helping as many deserving individuals as possible experience the value of DjangoCon US.
+
+
+These grants can assist with expenses such as:
+- **Travel costs**: Transportation to Chicago.
+- **Ground travel**: Getting to the conference to where you are staying.
+- **Accommodation**: Lodging during the conference.
+- **Meals**: Ensuring you have nourishment throughout the conference.
+- **Visa fees**: If applicable for international attendees.
+- **Other relevant expenses**: We understand that individual circumstances vary.
+
+The application process is straightforward and conducted through a Google form. We want to make it easy for you to express interest in attending DjangoCon US.
+
+The [form](https://docs.google.com/forms/d/e/1FAIpQLSfuriCI4JtAEcTfLy9Qdk2h_DfmaGKE544svRfMJLCOu5eLDQ/viewform) collects the following information to help us understand your situation and needs:
+- **Contact Information**: To communicate decisions.
+- **Affiliation**: Company, organisation, or individual.
+- **Django Background**: Your experience, reasons for attending, Django usage, and community involvement (to gauge interest).
+- **Attendance History**: First-time attendee status.
+- **Speaking Status**: If you've applied to speak.
+- **First-Time Speaker**: If accepted, would this be your first Django-related conference talk?
+- **Your Situation**: To understand your background.
+- **Financial Needs**: Funding amount (excluding ticket) and a cost breakdown (flights, travel, lodging, meals, visa, etc.).
+- **Payment Information**: Confirmation of PayPal or Wise account (flexible if needed).
```

Comment:
@adamfast do we want to push wise or should we mention another payment platform here?